### ISSUE TITLE
data.pop_usa -> Uncaught (in promise) ReferenceError: error is not defined

### ISSUE THREAD

**Issue by adamwiggins** [inkandswitch/livebook#64]
Steps to reproduce:

1. load welcome notebook
2. change "data.head()" to "data.year" (works as expected)
3. change "data.year" to "data.pop_usa" -- produces:

<img width="274" alt="screen shot 2016-02-01 at 21 44 43" src="https://cloud.githubusercontent.com/assets/177/12730635/1de92fe6-c92d-11e5-9269-1160c95e4be9.png">

Console says:

```
Uncaught (in promise) ReferenceError: error is not defined
    at http://livebook-staging.inkandswitch.com/js/worker.js:168:14
```

**Comment by adamwiggins** [comment]
Now seeing this in welcome notebook on first load

Steps to reproduce:

1. http://livebook-staging.inkandswitch.com/
2. do nothing, but live coding isn't working, check javascript console for the error

**Comment by adamwiggins** [comment]
Fixed by c94f2550fa4adcb22042686801db6cbfafa4d04d
